department-of-veterans-affairs/caseflow
# Add Dispatch test data to FACOLS

Resolves #5219 

Added the following test cases in cases.csv:
- A set of five amc_full_grants appeals.
- A set of five remands_ready_for_claims_establishment appeals.

### Acceptance Criteria 
 `RAILS_ENV=local rails db:setup` and
`RAILS_ENV=local rake local:vacols:seed` does not return any errors

### Testing Plan
1. Go to /dispatch/establish-claim/ and make sure there are 10 claims in the queue
2. Click on `Establish next claim`, then `Route claim`, then `Create End Product`
3. There should be `Success page` and text below `10 cases assigned, 1 completed`
